lib/loader/fs-resolver.js

```diff
@@ -90,7 +99,7 @@ FSResolver.prototype = {
       if (base) {
         local = path.relative(base, local);
       }
-      if (root) {
+      if (root && !isAbsolute(local)) {
```

Comment:
Well I didn't want to introduce a dependency on node 0.12 since current Ubuntu 15.04 ships with 0.10. That would have to include the npm package https://www.npmjs.com/package/path-is-absolute which felt too invasive for this.